jni/audioengine.cpp

```diff
@@ -42,33 +42,26 @@
 
 namespace AudioEngine
 {
-    int bytes_per_beat;
-    int bytes_per_bar;
-    int bytes_per_tick;
-
-    int amount_of_bars         = 1;
-    int beat_subdivision       = 4;
-    int min_buffer_position    = 0;  // initially 0, but can differ when looping specific measures
-    int max_buffer_position    = 0;  // calculated when sequencer API creates output
-    int marked_buffer_position = -1; // -1 means no marker has been set, no notifications will go out
-    int min_step_position      = 0;
-    int max_step_position      = 16;
-
     bool playing          = false;
     bool recordOutput     = false;
     bool haltRecording    = false;
     bool bouncing         = false;
     bool recordFromDevice = false;
-    bool monitorRecording = false; // might introduce feedback on microphone ;)
+    bool monitorRecording = false; // might introduce feedback when using internal microphone ;)
     int recordingFileId = 0;
 
-    /* buffer read/write pointers */
-
-    int bufferPosition = 0;
-    int stepPosition   = 0;
-
-    /* tempo related */
-
+    /* tempo / sequencer position related */
+
+    int samples_per_beat;                // strictly speaking sequencer specific, but scoped onto the AudioEngine
```

Comment:
Grouped all these values together for ease of reading. Also renamed for convenience (e.g. "bytes_per_x" is now "samples_per_x" to avoid confusion), renamed "tick" to "step".